src/TestSuite/Fixture/TestFixture.php

```diff
@@ -80,6 +80,13 @@ class TestFixture implements FixtureInterface
     protected $_schema;
 
     /**
+     * Fixture constraints to be created.
+     *
+     * @var array
+     */
+    public $constraints = [];
```

Comment:
Can we have a protected property instead of a public? Public properties always cause problems long term.